File: 2014/20140910 - miojo - javascript/miojo.js
exports.miojo = function(amp1, amp2) {
    if (amp1 == 3 || amp2 == 3) {
        return 3
    }

    if (amp1 == 1 || amp2 == 1) {
        return amp2 > amp1 ? amp2 : amp1
    }

    return 8
}

File: 2016/20160330 - miojo - python/miojo.py
def miojo(amp1, amp2):
	if (amp1 == 1 or amp2 == 1 or amp1 == 3 or amp2 == 3):
		return 3

	if(amp2 == 11):
		return 14

	if(amp1 == 7 or amp2 == 7):
		return 10

	return max(amp1, amp2)
